UnityPython.BackEnd/src/Traffy.Objects/List.cs

```diff
@@ -52,6 +53,12 @@ public static TrObject datanew(BList<TrObject> args, Dictionary<TrObject, TrObje
             throw new TypeError($"{clsobj.AsClass.Name}.__new__() takes 1 or 2 positional argument(s) but {narg} were given");
         }
 
+        public TrObject __add__(TrObject other)
+        {
+            append(this, other);
```

Comment:
No, python `list.__add__` does not change the self list. We can implement `__iadd__` later

Reply:
and `list.__add__` is in fact the concat of two lists, not adding one element.